src/mixins/notificationMixin.js

```diff
@@ -0,0 +1,36 @@
+export default {
+  methods: {
+    $notify(options) {
+      this.$store.commit(
+        'addNotification',
+        Object.assign({
```

Comment:
`Object.assign()` is not supported by IE.
Just make sure your babel configuration use a polyfill for `Object.assign()`.